aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsFluentClientDecorator.kt

```diff
@@ -74,7 +74,7 @@ private class AwsClientGenerics(private val types: Types) : FluentClientGenerics
     override val bounds = writable { }
 
     /** Bounds for generated `send()` functions */
-    override fun sendBounds(input: Symbol, output: Symbol, error: RuntimeType): Writable = writable { }
+    override fun sendBounds(operation: Symbol, operationOutput: Symbol, operationError: RuntimeType, retryPolicy: Any): Writable = writable { }
```

Comment:
It's hard to tell from the diff, but it looks like only `RustType`s are ever passed into these `Any` args. What's forcing them to be `Any`?

Reply:
For whitelabel clients, the passed-in type is `RustType.Unit`

Reply:
`RustType.Unit` is a `RustType` though?

Reply:
Ah, sorry, my brain rewrote your original comment to be about `RuntimeType`. `RuntimeType` and `RustType` are separate right?